src/part_b/CapGain.java

```diff
@@ -0,0 +1,125 @@
+/**
+ * @author snishi
+ * @date 7/10/2021
+ */
+
+package part_b;
+
+import java.util.Scanner;
+
+public class CapGain {
+    private double totCapGain;
+    private Queue sharesHeld;
+
+    public CapGain(){
+        totCapGain = 0.0;
+        sharesHeld = new Queue();
+    }
+
+    public void updateTotCapGain(double totCapGain){
+        this.totCapGain += totCapGain;
+    }
+
+    public double getTotCapGain(){
+        return totCapGain;
+    }
+
+    /**
+     * @about prompt for main menu items
+     */
+    public static void mainMenuPrompt(){
```

Comment:
this function is repeatedly calling println to add a new line, and needs refactoring